### ISSUE TITLE
Some Mister system symlinks disappeared.

### ISSUE THREAD

**Issue by totaldrk62** [danmons/retronas#359]
Some of my Mister symlinks have seemingly disappeared. The files still exist on the top level directory but the symlinks in the Mister game folders no longer exist. I've tried running the samba for mister script again but they aren't recreated. Oddly certain systems remain fine (MegaDrive for example) but others have no symlinks anymore (PSX, Megadrive CD). I don't believe I changed anything, just one day they stopped showing up. These had all been working previously and are now gone. Running on a Unraid VM with a mounted share which all seems to be running normally.

**Comment by sairuk** [comment]
i've rolled a new instance today and run the installer it is working as expected the links are created for all systems

Double check the update_all script hasn't created folders for the systems, if it has the symlinks wont be able to be created.

**Comment by totaldrk62** [comment]
Seems like a pretty typical structure to me.

![image](https://github.com/danmons/retronas/assets/110993967/3e3856f9-1a7b-46d0-8f74-b7f0072a9d07)

**Comment by totaldrk62** [comment]
Just a quick follow up. I went in and deleted the files in the PSX folder manually through the VM. I did a rebuild of samba for Mister and it still didn't take. I deleted the PSX folder itself and did a rebuild and for whatever reason that did the trick. Not quite sure why that worked but it did. I'm assuming the same resolution will work on the rest of the folders that seem to have issues. 

I'll update if for some reason that doesn't work but consider this one closed. Thanks for the help sairuk!
